web/src/pages/Route/List.tsx

```diff
@@ -343,6 +343,7 @@ const Page: React.FC = () => {
     },
     {
       title: formatMessage({ id: 'page.route.path' }),
+      dataIndex: 'uri',
```

Comment:
Actually, we use `uris` in the frontend 🤔

Reply:
I have done it according to the params , is it okay to do so?

Reply:
You need to convert uri to [uri] and uris

Reply:
Sorry, i didn't got you

Reply:
Hi @juzhiyuan.I check the backend code. We don't need to cover uri to [uri] as a query parameter.

The backend will check the URI in both `route.uri` and `route.uris`.